.circleci/config.yml

```diff
@@ -128,14 +128,14 @@ jobs:
       - run:
           name: Install modules and dependencies.
           command: |
-            npm install
+            npm install --unsafe-perm
```

Comment:
Pick up https://github.com/googleapis/nodejs-logging-winston/pull/25 instead.

Reply:
That change has been landed with #19 and @alexander-fenster has applied those changes here.